packages/client/package.json

```diff
@@ -74,14 +74,15 @@
     "@mui/icons-material": "^5.8.4",
     "@mui/material": "^5.9.2",
     "@sentry/browser": "^5.8.0",
-    "@tiptap/extension-character-count": "^2.0.0-beta.25",
-    "@tiptap/extension-link": "^2.0.0-beta.38",
-    "@tiptap/extension-mention": "^2.0.0-beta.102",
-    "@tiptap/extension-placeholder": "^2.0.0-beta.48",
-    "@tiptap/html": "^2.0.0-beta.209",
-    "@tiptap/react": "^2.0.0-beta.109",
-    "@tiptap/starter-kit": "^2.0.0-beta.185",
-    "@tiptap/suggestion": "^2.0.0-beta.97",
+    "@tiptap/extension-character-count": "^2.0.0-beta.218",
+    "@tiptap/extension-link": "^2.0.0-beta.218",
+    "@tiptap/extension-mention": "^2.0.0-beta.218",
+    "@tiptap/extension-placeholder": "^2.0.0-beta.218",
+    "@tiptap/html": "^2.0.0-beta.218",
+    "@tiptap/pm": "^2.0.0-beta.218",
```

Comment:
Note that this is a new package we need to add now that we have packages newer than [`2.0.0-beta.210`](https://github.com/ueberdosis/tiptap/blob/main/CHANGELOG.md#200-beta210-2023-02-02)